pkg/image/prune/prune.go

```diff
@@ -39,8 +39,12 @@ const (
 	// not keep an ImageNode from being a candidate for pruning.
 	WeakReferencedImageEdgeKind = "WeakReferencedImage"
 
+	// ReferencedImageConfigEdgeKind defines an edge from an ImageStreamNode or an
+	// ImageNode to an ImageComponentNode.
+	ReferencedImageConfigEdgeKind = "ReferencedImageConfig"
+
 	// ReferencedImageLayerEdgeKind defines an edge from an ImageStreamNode or an
-	// ImageNode to an ImageLayerNode.
+	// ImageNode to an ImageComponentNode.
 	ReferencedImageLayerEdgeKind = "ReferencedImageLayer"
```

Comment:
I don't mind the differentiation. The first comment was too quick, I've later noticed you've renamed the variable in the comment but all the rest is left. I'm just saying one should reflect the other.